.ci/install_kata_image.sh

```diff
@@ -13,20 +13,36 @@ set -o errtrace
 cidir=$(dirname "$0")
 rust_agent_repo="github.com/kata-containers/kata-containers"
 arch=$("${cidir}"/kata-arch.sh -d)
+TEST_INITRD="${TEST_INITRD:-no}"
+AGENT_INIT="${AGENT_INIT:-no}"
 
 build_rust_image() {
 	export RUST_AGENT="yes"
 	osbuilder_path="${GOPATH}/src/${rust_agent_repo}/tools/osbuilder"
 	distro="ubuntu"
 
-	pushd "${osbuilder_path}"
-	echo "Building rust image"
-	sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e image
-
 	image_path="/usr/share/kata-containers/"
 	sudo mkdir -p "${image_path}"
-	echo "Install rust image to ${image_path}"
-	sudo install -D "${osbuilder_path}/kata-containers.img" "${image_path}"
+
+	pushd "${osbuilder_path}"
+
+	if [ "${TEST_INITRD}" == "no" ]; then
+		echo "Building image with AGENT_INIT=${AGENT_INIT}"
+		sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e image
+
+		echo "Install image to ${image_path}"
+		sudo install -D "${osbuilder_path}/kata-containers.img" "${image_path}"
+	else
+		echo "Building initrd with AGENT_INIT=${AGENT_INIT}"
+		export ROOTFS_DIR="${osbuilder_path}/rootfs-${distro}"
+		sudo -E AGENT_INIT="${AGENT_INIT}" USE_DOCKER=true \
+			./rootfs-builder/rootfs.sh "${distro}"
+		sudo -E ./initrd-builder/initrd_builder.sh $ROOTFS_DIR
+
+		echo "Install initrd to ${image_path}"
+		sudo install -D "${osbuilder_path}/kata-containers-initrd.img" "${image_path}"
```

Comment:
Instead of using a literal I'd consider parameterising the link name (and also above for the image case) the way it's done in other scripts.  I'm quite sure this will need to be done at some point anyway (again, just as with other similar scripts) :-).

Reply:
Also, are all of the sudo's necessary?  I think `initrd_builder.sh` doesn't need root and I'd expect `rootfs.sh` not to need root either...